jenkins/src/main/resources/de/medavis/lct/jenkins/config/ConfigurationProfile/config.jelly

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="name" title="${%name}" description="${%name.description}">
+        <f:textbox/>
+    </f:entry>
+    <f:entry field="defaultProfile" title="${%default}" description="${%default.description}">
+        <f:checkbox/>
```

Comment:
Wenn zwei Profile aktiv sind, dann wird der falsche Fehlertext "Profile name must be unique" ausgegeben

Reply:
Good catch!